cmd/root.go

```diff
@@ -74,7 +75,7 @@ func Execute() error {
 }
 
 func doPrint(paths []string, fn func(*tfconf.Module) (string, error)) {
-	module, err := tfconf.CreateModule(paths[0])
+	module, err := tfconf.CreateModule(paths[0], settings)
```

Comment:
I would inject only the filename here. It's the only thing you're using really

Reply:
Oh oops, I forgot to remove that. Thanks for pointing it out!